CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Minor: `ref.value` for CustomFieldInputNumber now returns its type as Integer
```

Comment:
Theoretically, this is a breaking change so it would be a major. Usually, with breaking changes, it is recommended to add a quick note on how to upgrade.

In this case: Usages of `value` on the ref API can be interpolated in a string. i.e. `String(ref.current.value)`